recipes/rb-jekyll-compose/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "jekyll-compose" %}
+{% set version = "0.9.0" %}
+
+package:
+  name: rb-{{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://rubygems.org/downloads/{{ name }}-{{ version }}.gem
+  sha256: ebbf3bc13f9f8ad4d9268d28a379b37e3805ec8f2779209e61795217ea707a8e
+
+build:
+  noarch: generic
+  number: 0
+  script:
+    - gem install -N -l -V --norc --ignore-dependencies {{ name }}-{{ version }}.gem
+  skip: {{ win }}
+
+requirements:
+  host:
+    - ruby
+  run:
+    - ruby
+    - rb-jekyll >=3.0,<4
```

Comment:
I have been having issues when the Jekyll version spec allows it to install 4.0.0. 
I am investigating that separately... For the moment, we can get an older version of the package and wait for the auto-tick-bot to update it.